origin-dapp/src/components/wallet-card.js

```diff
@@ -61,14 +61,22 @@ class WalletCard extends Component {
       wallet: { address }
     } = this.props
 
-    getEthBalance()
-    getOgnBalance()
+    // getEthBalance()
+    // getOgnBalance()
+    this.pollBalance = setInterval(() => {
+      getEthBalance()
+      getOgnBalance()
+    }, 10000)
```

Comment:
Do we need to poll for balance here? I guess it's not a bad idea since a user could get ETH on an exchange, then come back and expect to see their balance updated without navigating or refreshing.